src/main/java/aitsi/m3spin/query/preprocessor/exceptions/UnknownAttributeNameException.java

```diff
@@ -0,0 +1,11 @@
+//package aitsi.m3spin.query.preprocessor.exceptions;
```

Comment:
Jeśli jest w całości zakomentowany, to czy plik raczej do wyrzucenia

Reply:
usunięty